lib/resources/postgres.rb

```diff
@@ -18,27 +19,22 @@ def initialize
         # Debian allows multiple versions of postgresql to be
         # installed as well as multiple "clusters" to be configured.
         #
-        version = version_from_dir('/etc/postgresql')
-        cluster = cluster_from_dir("/etc/postgresql/#{version}")
-        @conf_dir = "/etc/postgresql/#{version}/#{cluster}"
-        @data_dir = "/var/lib/postgresql/#{version}/#{cluster}"
+        @version = version_from_psql.to_s
```

Comment:
I suppose not, just being pedantic perhaps, ok we can remove it - I know it will return a string but just making sure. :)